### ISSUE TITLE
Dialogs are not clickable on computers with touch screen

### ISSUE THREAD

**Issue by furushchev** [phonon-framework/phonon#276]
Hi, I use a phonon framework as web application for pc / smartphone.
This framework is mostly really nice, however in dialogs any buttons do not work for PCs w/o touch devices maybe because they are not clickable.

```js
var prompt = phonon.confirm("Hey");
prompt.on("confirm", function(value) {
 console.log("confirm clicked");  // this is never displayed on click in PCs. It works well on smartphone with touch devices.
});
```

Other buttons that are hooked via `object.on('tap', function(){...})` also don't work, though  `object.on('click', function(){...})` works on click in PCs.

**Comment by John-Henrique** [comment]
Here works perfectly. Can you try use this in your PC? http://phonon.quarkdev.com/components/ navigate to dialogs...

**Comment by furushchev** [comment]
@John-Henrique 
Thank you very much for reply and sorry for late reply!
I further investigated and found that if the computer has a touch screen, then all buttons become unclickable...

**Comment by furushchev** [comment]
I changed the title of this issue.
Also found that it occurs also with Firefox.

My environment is:
- OS: Ubuntu 14.04.5 / 16.04.3
- Browser:
  - Google Chrome 
  - Firefox 57.0.4 (64-bit)

**Comment by furushchev** [comment]
Any proceedings?

**Comment by qathom** [comment]
Hi @furushchev,

It seems that `user-select` is causing this problem.
Phonon v1.5.1 will remove this property by default.

**Comment by furushchev** [comment]
@qathom Thank you for the information! It should fix the issue!

**Comment by qathom** [comment]
Thanks @furushchev 
I just published v1.5.1 :)
